PowerShellTools/DebugEngine/HostUi.cs

```diff
@@ -225,6 +230,72 @@ public string ReadLine(string message)
         }
 
         /// <summary>
+        /// Ask for securestring from user
+        /// </summary>
+        /// <param name="message">Message of dialog window.</param>
+        /// <param name="name">Name of the parameter.</param>
+        /// <returns>A PSCredential object that contains the securestring.</returns>
+        public async Task<PSCredential> ReadSecureStringAsPSCredential(string message, string name)
+        {
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+
+            SecureString secString = new SecureString();
+            SecureStringDialogViewModel viewModel = new SecureStringDialogViewModel(message, name);
+            SecureStringDialog dialog = new SecureStringDialog(viewModel);
+
+            var ret = dialog.ShowModal();
+            if(ret.HasValue && ret.Value == true)

```

Comment:
ShowModal return bool? instead of bool, so you need to check hasvalue before casting it into boolean.